conjur-openapi.yml

```
@@ -221,6 +223,30 @@ components:
       description: "Example of a Conjur API key"
       example: "14m9cf91wfsesv1kkhevg12cdywm2wvqy6s8sk53z1ngtazp1t9tykc"
 
+    Authenticators:
+      type: object
+      properties:
+        installed:
+          description: "The authenticators installed on the Conjur server"
+          type: array
+          items:
+            type: string
+          example: [ "authn" ]
+
+        configured:
+          description: "The authenticators configured on the Conjur server"
+          type: array
+          items:
+            type: string
+          example: [ "authn" ]
+
+        enabled:
+          description: "The enabled authenticators on the Conjur server"
```

```suggestion
          description: "The authenticators enabled on the Conjur server"
```

should be consistent with lines above